app/react/common/state/roles/roles.actions.js

```diff
@@ -0,0 +1,24 @@
+import RolesEndpoints from "react/common/endpoints/roles";
+
+export const REQUEST_ROLES = "REQUEST_ROLES";
+export const RECEIVE_ROLES = "RECEVIE_ROLES";
```

Comment:
happened to see a typo here, didn't want to ignore it

Reply:
How was this working with the typo?!?